galite-core/src/main/kotlin/org/kopi/galite/form/dsl/FormBlock.kt

```diff
@@ -192,10 +203,10 @@ open class FormBlock(var buffer: Int, var visible: Int, ident: String, val title
   /**
    * Initializes a field.
    */
-  inline fun <reified T: Comparable<T>> initField(domain: Domain<T>,
-                                                  init: FormField<T>.() -> Unit,
-                                                  access: Int,
-                                                  position: FormPosition? = null): FormField<T> {
+  inline fun <reified T : Comparable<T>> initField(domain: Domain<T>,
```

Comment:
The part to add the field with droppable is implemented insed the initField method with this part in line 1868 in FormParser.java  , but you will need to find alternatives for VKstringtype , VKImageType and reportTrouble() function  , afterwards the implementation will look like this : 

```javascript
    inline fun <reified T : Comparable<T>> initField(domain: Domain<T>,
                                                       init: FormField<T>.() -> Unit,
                                                       access: Int,
                                                       position: FormPosition? = null): FormField<T> {
        domain.kClass = T::class
        val field = FormField(this, domain, blockFields.size, access, position)
        field.init()
        field.setInfo()
    
        if (dropList == null) {
          blockFields.add(field)
        } else {
          if (type.getDef() !is VStringType
                  && type.getDef() !is VKImageType) {
            reportTrouble(PositionedError(sourceRef,
                    FormMessages.UNSUPPORTED_DROP_FIELD_TYPE,
                    name))
          } else {
            val flavor: String? = this.addDropList(dropList!!, field)
            if (flavor == null) {
              blockFields.add(field)
            } else {
              reportTrouble(PositionedError(sourceRef,
                      FormMessages.DUPLICATE_DROP_FLAVOR,
                      flavor,
                      this.dropListMap[flavor]))
            }
          }
        }
        return field
      }
```